src/pages/Cart.js

```diff
@@ -35,109 +35,111 @@ export const Cart = () => {
 
   return (
     <>
-      {productsInCart.length ? (
-        <div className="cart-layout nav-adjust">
-          <div className="cart">
-            <div className="cart-heading">
-              <h3>product</h3>
-              <h3>quantity</h3>
-              <h3>price</h3>
-            </div>
-            {productsInCart.map(
-              ({
-                _id,
-                name,
-                imgUrl,
-                description,
-                price,
-                discount,
-                quantity
-              }) => (
-                <div className="product-card" key={_id}>
-                  <div
-                    className="product-details"
-                    style={{ cursor: "pointer" }}
-                    onClick={() => navigation(`/product/${_id}`)}
-                  >
-                    <img className="product-img" src={imgUrl} alt="product" />
-                    <div className="wrapper">
-                      <h3 className="product-title">
-                        {name}
-                        <span> 1kg</span>
-                      </h3>
-                      <h4 className="product-price">
-                        ₹{(price - (price * discount) / 100).toFixed(2)}
-                        <span className="strike">₹ {price}.00</span>
-                      </h4>
-                      <h4 className="product-discount">
-                        Discount: {discount}%
-                      </h4>
-                      <p className="product-description">{description}</p>
-                      <p className="d-block d-sm-none"></p>
+      <div className="cart-layout nav-adjust">
+        {productsInCart.length ? (
+          <>
+            <div className="cart">
+              <div className="cart-heading">
+                <h3>product</h3>
+                <h3>quantity</h3>
+                <h3>price</h3>
+              </div>
+              {productsInCart.map(
+                ({
+                  _id,
+                  name,
+                  imgUrl,
+                  description,
+                  price,
+                  discount,
+                  quantity
+                }) => (
+                  <div className="product-card" key={_id}>
+                    <div
+                      className="product-details"
+                      style={{ cursor: "pointer" }}
```

Comment:
Can add it in css

Reply:
Hey i have completed the mentioned changes.